python/deadlinesubmissionsettings/lib.py

```diff
@@ -29,21 +51,21 @@ def query(argument):
 
 def get_machine_list(debug=None):
     """Fetch the machine list (slaves) from Deadline"""
-    AVALON_SESSION = debug or avalon.Session["AVALON_DEADLINE"]
+    AVALON_SESSION = debug or api.Session["AVALON_DEADLINE"]
```

Comment:
Does it make more sense to call this variable `avalon_deadline` or something? It's not actually the session. :)

Reply:
LOL, this slipped in without me even noticing. I know I wanted to call it AVALON_DEADLINE :')